src/github-writer.css

```diff
@@ -28,11 +28,11 @@ div.github-writer-ckeditor .ck-editor__editable {
 }
 
 div.github-writer-ckeditor .ck-editor__editable.ck-blurred {
-	background-color: #fafbfc;
+	background-color: transparent;
```

Comment:
Just to put my 2 cents in here I see that the initial idea was to make it darker when it's blurred (a feature that will be gone when giving transparent\` background in bot cases).

The background color could be set in rgba format `rgba( 0, 0, 0, 0.2 )` (an example) to not have any particular color hardcoded.

Reply:
That alpha value is a good way to have one solution for both light/dark themes. 

But it doesn't look like Github changes the background color when the field is blurred (they may have in the past). If that's the case, I would recommend inheriting their same UX and using a transparent background for both blurred/focused states.

Reply:
> But it doesn't look like Github changes the background color when the field is blurred

GitHub indeed changes the background color when blurring text fields. The change is minimal and it happens in both light and dark modes:

 * Light Theme: on focus - totally white / on blur - light gray
 * Dark Theme: on focus - almost black gray / on blur - totally black

Reply:
@trackwell-mike, if you don't mind, I'm taking over this one, based on your PR. The solution is pretty simple and is similar to the other changes you proposed. It's enough to use CSS variables now provided by GitHub itself and the trick is done. I'll come with a new PR for it (including your original changes).

Thanks for the contribution!

Reply:
By all means! Thanks for responding to this issue. 

There are some additional considerations as well. For example when you mention someone, on the dark theme, the font color needs to be inherited/changed based on Github's native UI.

Reply:
Indeed. All this is in fact covered by #241. Thanks again ;)